tests/remote/check_alert_source_usage/variables.tf

```diff
@@ -1,32 +1,9 @@
 variable "alert_name_predicate" {
   type = object({
     type  = string
-    value = string
+    value = optional(string)
   })
   description = "A condition that should be satisfied."
-
-  validation {
```

Comment:
This validation has been moved into the provider code and is no longer needed here